PACKAGE='robosim_gazebo'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
# Wheels
gen.add("wheel_size_r", double_t, 2, "wheel size r", 0, 0.01, 4)
gen.add("wheel_size_d", double_t, 2, "wheel size d", 0, 0.01, 4)
# Body
gen.add("body_size_x", double_t, 2, "robot body size x", 0, 0.01, 4)
gen.add("body_size_y", double_t, 2, "robot body size y", 0, 0.01, 4)
gen.add("body_size_z", double_t, 2, "robot body size z", 0, 0.01, 4)
gen.add("body_pos_x", double_t, 2, "body pos x", 0, -4, 4)
gen.add("body_pos_y", double_t, 2, "body pos y", 0, -4, 4)
gen.add("body_pos_z", double_t, 2, "body pos z", 0, -4, 4)
# Caster
gen.add("caster_size_r", double_t, 2, "caster size r", 0, 0.01, 4)
gen.add("caster_pos_x", double_t, 2, "caster pos x", 0, -4, 4)
gen.add("caster_pos_y", double_t, 2, "caster pos y", 0, -4, 4)
gen.add("caster_pos_z", double_t, 2, "caster pos z", 0, -4, 4)

exit(gen.generate(PACKAGE, "robosim_gazebo", "Robosim"))